visualizations/frontend/other/TTable.vue

```diff
@@ -72,6 +72,7 @@
       id="tableContainer"
       ref="tableContainer"
       :style="columnWidthsStyle"
+      :class="extraClass"
```

Comment:
Have you tried this out? Is there an associated branch in the reporting that I could see how you're using this?

Reply:
Yes, please checkout to the feat/enterprise_reporting branch of the topcoat-reports repo. I just pushed the code for testing.